analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -718,7 +721,8 @@ def signal_handler(signum, frame):
     analyzed_actions = [(actions_map,
                          build_action,
                          context,
-                         analyzer_config_map,
+                         analyzer_config_map.get(build_action.analyzer_type,
```

Comment:
By default [get](https://docs.python.org/3/library/stdtypes.html#dict.get) will return with None if the key is not found in the map.